src/plugins/dragresize_ie11.js

```diff
@@ -485,9 +485,16 @@
 				if (
 					editor.filter.checkFeature(this.features.dimension) &&
 					editor.config.ae_dragresize_ie11_disableResizer !== true
-				)
+				) {
 					setupResizer(this);
 
+					var dragHandlerStyle = this.dragHandlerContainer.$.style;
+					dragHandlerStyle.setAttribute('backgroundColor', 'rgba(255, 255, 255, 1');
+					dragHandlerStyle.setAttribute('opacity', '1');
+					dragHandlerStyle.setAttribute('margin-top', '-5px');
+					dragHandlerStyle.setAttribute('margin-left', '-20px');
```

Comment:
Wondering if this would belong better inside `setupResizer()`? I don't know if you consider it to be related to that, but it looks that way to me.

Reply:
I think it's technically separate so maybe I should move it out of the if block since I'm not sure that the resizing and the drag and drop handle are explicitly linked functionality